.github/workflows/release.yaml

```diff
@@ -24,3 +24,10 @@ jobs:
             _output/components.yaml
             _output/high-availability.yaml
             _output/high-availability-1.21+.yaml
+      - name: Build binaries
+        run: make release-binaries
+      - name: Release
+        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # v0.1.15
+        with:
+          files: |
+            _output/metrics-server-*
```

Comment:
The build process takes approximately 10 minutes. Avoid delaying the release of the manifest.